cmd/terramate/cli/cli.go

```diff
@@ -812,11 +802,20 @@ func (c *cli) runOnStacks() {
 		return
 	}
 
-	logger.Debug().
-		Msg("Run command.")
-	err = terramate.Run(c.root(), order, cmd)
+	logger.Trace().Msg("Get command to run.")
+	cmd := run.Cmd{
+		Path:    c.parsedArgs.Run.Command[0],
+		Args:    c.parsedArgs.Run.Command[1:],
+		Stdin:   c.stdin,
+		Stdout:  c.stdout,
+		Stderr:  c.stderr,
+		Environ: os.Environ(),
+	}
+
+	logger.Debug().Msg("Run command.")
+	err = run.Run(c.root(), order, cmd)
```

Comment:
```suggestion
	err = cmd.Run(c.root(), order)
```

Probably because I'm used to Go's exec.Command API =P. But feel free to disregard.